mantidproject/mantid
# Update MacOS dmg background image

Description of work.

This updates the MacOS installer to show a background image. It uses the same script discussed in the [Kitware Blog](https://blog.kitware.com/creating-mac-os-x-packages-with-cmake/).

**To test:**

Download .dmg from the build servers and check that it looks like the image below

![mantid installer](https://cloud.githubusercontent.com/assets/5920552/20498078/bbc7a82e-aff9-11e6-9bc3-93d2f7f484ac.png)



There is no GitHub issue associated with this pull request. 

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
